setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = django-sorting-field
-version = 1.0.0.post+gitver
+version = 1.0.1.post+gitver
```

Comment:
Don't bump version numbers in PRs. The release process should be separate. This also breaks the automatic versioning by gitver, since it expects there to be a tag named `X`, if, the version is `X.post+gitver`.

Reply:
I see. will do so next time